### DISCUSSION TITLE
Warning: win10,  System locale is not 'UTF-8'!

### DISCUSSION THREAD

**Discussion by UFOAlastor** [houtianze/bypy#626] [Q&A]
<W> WARNING: System locale is not 'UTF-8'.
Files with non-ASCII names may not be handled correctly.
You should set your System Locale to 'UTF-8'.
Current locale is 'cp65001'

**Comment by RebelYoung** [comment]
Yes, in win10, I have change the location setting into Beta version, but the warning still existed and command were useless.
![image](https://user-images.githubusercontent.com/59256512/216824866-db76b1af-d605-47a5-a520-b6b944d94ab9.png)

**Reply by Waves-Rim** [reply]
So, does bypy only work on linux?
